docs/registry/configure.md

```diff
@@ -206,26 +159,34 @@ The custom model is configured as a component with the name "my-realsense".
 {{% /tab %}}
 {{% /tabs %}}
 
-### Edit the module
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< alert title="Note" color="note" >}}
```

Comment:
@npentrel is it okay if I delete this note? I feel like it belongs in the try viam docs, not here